README.md

```diff
@@ -18,26 +22,31 @@ const MY_SERVICE_META = {
   multiArg: false, // defaults to false. If true accepts an array for arguments, if false an array will be assumed to be the first (and only) argument.
   expose: [
     // The methods to be exposed publicly
-    // BREAKING CHANGE: expose now requires an object
-    { methodName: "doStuff" },
+
     {
       methodName: "moreStuff",
       methodTimeout: 15000,
       paramNames: ["stuffs"],
-      help: "This is a silly method"
-    }
-  ]
+      help: "This is a silly method",
+    },
+  ],
 };
 
-const myRpcService = lokeHttpRpc.createRequestHandler(
-  myService,
-  MY_SERVICE_META
-);
+//BREAKING CHANGES:
+// "createRequestHandler" was renamed to createWellKnownHandler and can be accesed through register()
+// Added new createWellKnownHandler which serves the metadata
+// WELL_KNOWN_META_PATH represents the path where meta data served
```

Comment:
Don't do this as comments. Put this right at the top as a heading, eg

```
## Breaking Changes for v5

Services need to be registered before a request handler can be created.

### Migrating from v4 to v5

... add before/after
```

Reply:
done